sections/semantics-sections.include

```diff
@@ -1397,7 +1440,7 @@
     There are currently no known implementations of the outline algorithm in graphical browsers or
```

Comment:
Hmm. There are extensions that do it, there are just no native outline implementations I am aware of. It's an important difference.